taichenl/571G_proj
# Please merge

Tested features:
1. create a new user with a meta mask wallet address
2. view results of a poll
3. create a poll with hardcode input

TODOs:
1. Development:
1.1 Merge in the content of the poll creation board and then use user input to replace hardcode input for poll creation.
1.2 We are not able to grab poll selections from our smart contract, for some reason, the options array is null/undefined at this moment. A possible solution could be rephrasing the array to a string and then passing the string from the smart contract to the web3 handler. And then split the string by Javascript before rendering it to the UI component.

2. Testing:
2.1 Voting should be done successfully. 
2.2 Voting should be correctly represented by the results.
